cli/pom.xml

```diff
@@ -174,6 +174,33 @@
 
   <profiles>
     <!-- Builds the native image and archives it with the documentation from the github artifact -->
+    <profile>
+      <id>assembly</id>
+      <build>
+        <plugins>
+          <!-- Assembly Plugin -->
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-assembly-plugin</artifactId>
+            <version>${assembly.maven.plugin.version}</version>
+            <executions>
+              <execution>
+                <id>create-distribution</id>
+                <phase>install</phase>
+                <goals>
+                  <goal>single</goal>
```

Comment:
dont we have to change this now? With the new design we need to build all releases in one maven run now so we need to tell assembly plugin to build all assemblies in a "loop".

Reply:
Can be closed as it is still a single process, using `descriptorSourceDirectory` now though.

Reply:
This was a misunderstanding on my end.
I was expecting the `single` goal to process a single Assembly XML file. However, it already builds all Assembly XML files since we configure the directory containing them.